### DISCUSSION TITLE
Problem in converting pdb to extxyz format and training

### DISCUSSION THREAD

**Discussion by KellekKan** [mir-group/nequip#348] [Q&A]
Q1.) The trajectory of the MD simulations has been computed by using AMBER, could anyone please tell me that how can I convert the pdb format to the extxyz format to be the input of the training data ? 

I used the following code but it is not possible to convert to the extxyz format with the error message shown (NameError: name 'extxyz' is not defined)

_import numpy as np
from ase.io import read, write
atoms = read('input.pdb')
write('output.extxyz', atoms, format='xyz')_

Q2.) Is it also possible for you to share me an input data in the extxyz format, and how can i arrange it for being the training set data ?

Thanks!

**Comment by Linux-cpp-lisp** [comment]
ASE does appear to support PDB: https://wiki.fysik.dtu.dk/ase/ase/io/formatoptions.html#proteindatabank

You can easily generate example extXYZ files using some combination of `ase.build` and `ase.io.write`: https://wiki.fysik.dtu.dk/ase/ase/io/io.html
